### ISSUE TITLE
Type of association reference list is lower case in UmpleOnline when clicked on "show tabs on different tabs"

### ISSUE THREAD

**Issue by amidzak** [umple/umple#970]
Considering the following Umple code:
```java
class D{
}
class C{
        * -- * D;
}
```
The following code is generated
```java
public class C
{
 
  //------------------------
  // MEMBER VARIABLES
  //------------------------
 
  //C Associations
  private List<d> ds;
 
  //------------------------
  // CONSTRUCTOR
  //------------------------
 
  public C()
  {
           ds = new ArrayList<d>();
```
`List<d>` is wrong it should be `List<D>`.
This problem seems to be a occurring only in the online version.

**Comment by TimLethbridge** [comment]
What is the problem? You say List should be List. ArrayList is correct as it is an implementor of List.

**Comment by amidzak** [comment]
Sorry. List\<d> should be List\<D>. It considered that as an HTML tag. I edited the issue.

**Comment by TimLethbridge** [comment]
I don't get this when I try in Umpleonline. Give a screenshot?

**Comment by TimLethbridge** [comment]
You are simply saying that the D appears lowercase when you try it?

**Comment by TimLethbridge** [comment]
Aha! I see it now when I click 'Show Files in Separate Tabs.

**Comment by TimLethbridge** [comment]
This is a bizarre issue. Not related, I think, to the actual generator. Something is translating when rendering the text in html. I will tag this as an UmpleOnline issue.

**Comment by amidzak** [comment]
I Changed the title

**Comment by Nava2** [comment]
I fixed the code formatting. I removed the `\` characters, hopefully they weren't actually showing up. Also, @amidzak we can see when the title changes 😄.

**Comment by TimLethbridge** [comment]
If I do this

```
// Test <D>
class Dd{ }
class C{ * -- * Dd; }
```
It also lowercases the D in the comment. So clearly it is the code that converts anything in <> when splitting into multiple files.

**Comment by Nava2** [comment]
I happen to have fought this issue a few days ago with my course website. I've proposed a fix in #972, please verify that it is working before merging.
